pages/api/study/index.js

```diff
@@ -0,0 +1,63 @@
+import { getToken } from 'next-auth/jwt'
+import dbConnect from '@lib/db-connect'
+import { generateKey } from '@lib/utils'
+import Study from '@models/study'
+import ParticipantData from '@models/participantData'
+
+export default async function handler(req, res) {

```

Comment:
Should also split this into 2 files.
- /api/study/[id].js for GET request
- /api/study for POST request